File: src/main/java/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/Activity.java
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.github.dqualizer.dqlang.types.domain_architecture_mapping.Endpoint;
import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

@Getter
@ToString
public class Activity {

    @JsonProperty("dq_id")
    private String dqID;
    private String name;
    @JsonProperty("operation_id")
    private String operationID;
    private String type;
    private LinkedHashSet<String> parameter;
    private Endpoint endpoint;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/Activity.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class Activity {
    @JsonProperty("dq_id")
    val dqID: String? = null
    val name: String? = null

    @JsonProperty("operation_id")
    val operationID: String? = null
    val type: String? = null
    val parameter: LinkedHashSet<String>? = null
    val endpoint: io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping.Endpoint? = null
}
